index.html

```diff
@@ -6,8 +6,150 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Live Concerts</title>
     <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.0/css/line.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
+    <header>
+      <div class="social-links">
+        <a href="#"><i class="uil uil-facebook"></i></a>
+        <a href="#"><i class="uil uil-twitter-alt"></i></a>
+        <a href="#">English</a>
+        <a href="#">Page</a>
+        <a href="#">Logout</a>
+      </div>
+      <div class="hamburger">
+        <button id="hamburger"><i class="uil uil-bars"></i></button>
+        <div class="nav-menu">
+          <div class="desktop-logo">
+            <a href="./index.html"
+              ><img src="./images/white-logo1.jpg" alt="live-concerts-logo"
+            /></a>
+          </div>
+          <div class="menu-list">
+            <img
+              class="close"
+              src="./images/menu-cross.png"
+              alt="close button"
+            />
+            <ul class="about-menu">
+              <li class="close"><a href="./about.html">About</a></li>
+              <li class="close"><a href="#events">Events</a></li>
+              <li class="close"><a href="#">Join</a></li>
+              <li class="close"><a href="#">Sponsor</a></li>
+              <li class="close"><a href="#">News</a></li>
+              <li class="book-now close"><a href="#"> Book Now</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <main>
+        <div class="main-container">
+          <h2>"South Africa's best marketplace for live events"</h2>
+          <h1>
+            LIVE CONCERTS <br />
```

Comment:
- [ ] Please make this heading larger as seen in the original design and add `mix-blend-mode: multiply;` so that the heading is transparent.
### Your Design
![image](https://user-images.githubusercontent.com/45460192/198294348-afa94854-f251-4062-8c89-e0d139001e05.png)
### Original
![image](https://user-images.githubusercontent.com/45460192/198294443-2482c646-21e1-403a-9b51-686c3893faa7.png)